code/modules/food/kitchen/cooking_machines/_cooker_output.dm

```diff
@@ -1,72 +1,160 @@
+	if (!prefix)
+		if (scale == min_scale)
+			prefix = "tiny"
+		else if (scale <= 0.8)
+			prefix = "small"
+
+		else
+			if (scale >= 1.2)
+				prefix = "large"
+			if (scale >= 1.4)
+				prefix = "extra large"
+			if (scale >= 1.6)
+				prefix = "huge"
+			if (scale >= max_scale)
+				prefix = "massive"
+
+		name = "[prefix] [name]"
+
+
+/obj/item/weapon/reagent_containers/food/snacks/variable/pizza
+	name = "personal pizza"
+	desc = "A personalized pan pizza meant for only one person."
+	icon_state = "personal_pizza"
+	size = 20
+	w_class = 3
+
+/obj/item/weapon/reagent_containers/food/snacks/variable/bread
+	name = "bread"
+	desc = "Tasty bread."
+	icon_state = "breadcustom"
+	size = 40
+	w_class = 3
+
+/obj/item/weapon/reagent_containers/food/snacks/variable/pie
+	name = "pie"
+	desc = "Tasty pie."
+	icon_state = "piecustom"
+	size = 25
+
+/obj/item/weapon/reagent_containers/food/snacks/variable/cake
+	name = "cake"
+	desc = "A popular band."
+	icon_state = "cakecustom"
+	size = 40
+	w_class = 3
+
+/obj/item/weapon/reagent_containers/food/snacks/variable/pocket
+	name = "hot pocket"
+	desc = "You wanna put a bangin- oh, nevermind."
+	icon_state = "donk"
+	size = 8
+	w_class = 1
+
+/obj/item/weapon/reagent_containers/food/snacks/variable/kebab
+	name = "kebab"
+	desc = "Remove this!"
+	icon_state = "kabob"
+	size = 10
+
+/obj/item/weapon/reagent_containers/food/snacks/variable/waffles
+	name = "waffles"
+	desc = "Made with love."
+	icon_state = "waffles"
+	size = 12
+
+/obj/item/weapon/reagent_containers/food/snacks/variable/cookie
+	name = "cookie"
+	desc = "Sugar snap!"
+	icon_state = "cookie"
+	size = 6
+	w_class = 1
+
+/obj/item/weapon/reagent_containers/food/snacks/variable/donut
+	name = "filled donut"
+	desc = "Donut eat this!" // kill me
+	icon_state = "donut"
+	size = 8
+	w_class = 1
+
+/obj/item/weapon/reagent_containers/food/snacks/variable/jawbreaker
+	name = "flavored jawbreaker"
+	desc = "It's like cracking a molar on a rainbow."
+	icon_state = "jawbreaker"
+	size = 4
+	w_class = 1
+
+/obj/item/weapon/reagent_containers/food/snacks/variable/candybar
+	name = "flavored chocolate bar"
+	desc = "Made in a factory downtown."
+	icon_state = "bar"
+	size = 6
+	w_class = 1
+
+/obj/item/weapon/reagent_containers/food/snacks/variable/sucker
+	name = "flavored sucker"
+	desc = "Suck, suck, suck."
+	icon_state = "sucker"
+	size = 4
+	w_class = 1
+
+/obj/item/weapon/reagent_containers/food/snacks/variable/jelly
+	name = "jelly"
+	desc = "All your friends will be jelly."
+	icon_state = "jellycustom"
+	size = 8
+
+
+/obj/item/weapon/reagent_containers/food/snacks/variable/cereal
+	name = "cereal"
+	desc = "Crispy and flaky"
+	icon_state = "cereal_box"
+	size = 30
+	w_class = 3
+
+/obj/item/weapon/reagent_containers/food/snacks/variable/cereal/Initialize()
+	. =..()
+	name = pick(list("flakes", "krispies", "crunch", "pops", "O's", "crisp", "loops", "jacks", "clusters"))
+
+/obj/item/weapon/reagent_containers/food/snacks/variable/mob

```

Comment:
This is unobtainable from what I can gather, and I don't think it interacts with holders in any way, so recipies, both regular and modular, involving this aren't a thing.

Reply:
Again, _appliance.dm line 648: `var/obj/item/weapon/reagent_containers/food/snacks/variable/mob/result = new /obj/item/weapon/reagent_containers/food/snacks/variable/mob(CI.container)`

Reply:
Nevermind, the diffs on that file just failed to open when I was making the comment. I see it now.